docs/kyma/docs/04-04-gke-installation.md

```diff
@@ -223,7 +229,7 @@ Follow these steps:
     kubectl -n kyma-installer logs -l 'name=kyma-installer'
     ```
 
-## Add the xip.io self-signed certificate to your OS trusted certificates
+## (Optional) Add the xip.io self-signed certificate to your OS trusted certificates
```

Comment:
I don't like the idea of putting a word in parenthesis in the heading. We have notes that state that some steps are optional. Additionally, I think we're going to restructure these docs using panels.

Reply:
if we later do the tabs with different options, awesome, not sure if with these sections it is possible, DNS (if you want to take this step) must be done upfront, while the step about xip at the end. Tricky case